### ISSUE TITLE
How to fix the problem core dumped ??

### ISSUE THREAD

**Issue by GhofraneHADHRI** [davheld/GOTURN#70]
when i try to run the train.sh i got this error 
![msg](https://user-images.githubusercontent.com/36658813/39138592-363da28c-4718-11e8-8801-33998b75d195.png) can anyone help me ???

**Comment by RJVisee44** [comment]
@GhofraneHADHRI 
Any chance you fix this? Running into same issue

**Comment by GhofraneHADHRI** [comment]
@RyanCodes44  yes I have solved this problem, it is a problem of path. In fact, I put the contents of the ImagNet folder in another folder to detect it.
So, instead of:
/ Media / Ghofrane / FAF2E7E7F2E7A65B / base / imagNet / ILSVRC2013_val
The path has become:
/ Media / Ghofrane / FAF2E7E7F2E7A65B / base / imagNet / images / ILSVRC2013_val
Similarly for the gt folder.
